suspect/processing/channel_combination.py

```diff
@@ -19,3 +24,125 @@ def combine_channels(data, weights=None):
     weighted_data = weights.reshape((len(weights), 1)) * data
     combined_data = weighted_data.sum(axis=0)
     return combined_data
+    
+
+def RAMRS_SVD_weighting(data, phase_correct = True, whiten = False, W = None, noise_range = None):
+    """
+    This channel combination approach operates on raw MRS (spectral) data, 
+    prior to averaging repetitions from multiple acquisitions. It is useful in 
+    cases where there is instability in frequency and phase shifts across 
```

Comment:
Interesting that we both thought about this in the same way and came to opposite conclusions. I am trying quite hard to maintain an easy to understand set of example notebooks on readthedocs, there is already a notebook on channel combination that looks at different ways of weighting the channels including the SVD method. My plan was to either add another notebook later in the series to cover noise whitening, or to go back and incorporate that into the existing notebook. The great thing about the notebooks is that they can combine explanation and plots inline with the code, so they help with understanding the concepts much better than simple comments. Although your docstring is excellent, most end users aren't going to dig through the codebase to understand how the algorithm works, they just want some functions they can call.
I think that a chunk of the code inside this function could do with being extracted into a `whiten()` function which can be called separately. Ultimately I think that the implementation of this function should be essentially just calling about 3 other functions from the module, and that **may** make it superfluous. If you don't have the time to work on that, then I am happy to take it on.